test/isArray_test.js

```diff
@@ -0,0 +1,57 @@
+goog.module('datalayerhelper.helper.testing.isArray_');
+goog.setTestOnly();
+
+const {isArray_} = goog.require('helper');
+
+describe(`The 'isArray_' function of helper`, () => {
```

Comment:
I'd generally prefer quotes around the main description and ticks around the function. Since people generally use quotes for descriptions, it is easier to just add function names (by using ticks) to descriptions which don't have them. Does that make sense?

Reply:
That seems fair, I will fix it.